File: android/src/main/java/com/bluecall/sinch/CallDelegate.java
package com.bluecall.sinch;

/**
 * Created by tamarabernad on 2017-09-11.
 */

public interface CallDelegate {
    void didReceiveCall(String callId);
    void callEndedWithReason(String reason, int duration);
    void callDidEstablish();
    void callDidProgress();
    void callDidChangeStatus(String status);
}

File: ios/Classes/CallDelegate.swift
//
//  CallDelegate.swift
//  Blue Call
//
//  Created by Tamara Bernad on 2017-06-22.
//  Copyright © 2017 Blue Call. All rights reserved.
//

import Foundation
@objc protocol CallDelegate{
    func callDidEnd(reason:String, duration:Double)
    func callDidEstablish()
    func callDidProgress()
    func callDidChangeStatus(status:String)
}
